src/ProcessNormalization.cc

```diff
@@ -93,7 +94,7 @@ Double_t ProcessNormalization::evaluate() const {
             const RooAbsReal *theta = asymmThetaListVec_.at(i);
             const std::pair<double,double> logKappas = logAsymmKappa_.at(i);
             double x = theta->getVal();
-            logVal +=  x * logKappaForX(x, logKappas);
+            logVal +=  x * logKappaForX(x, logKappas.second, logKappas.first);
```

Comment:
Can you not just call `logKappa` here, given that all `logKappaForX` does is call `logKappa`?